### PR TITLE
Content/61 add publications page

### PR DESCRIPTION
**Pull request by yaxue1123**

### CODE REVIEW

File: src/data/resources/publications/01-fabric-network-infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "FABRIC: A National-Scale Programmable Experimental Network Infrastructure"
+authors: "Ilya Baldin, Anita Nikolich, James Griffioen, Indermohan Inder S. Monga, Kuang-Ching Wang, Tom Lehman, Paul Ruth"
+link: https://ieeexplore.ieee.org/document/8972790
+doilink: https://doi.org/10.1109/MIC.2019.2958545
```
Comment (mbwatson) [inline src/data/resources/publications/01-fabric-network-infrastructure.md:5]: this field may be redundant. if the DOI is known, then the DOI link is always `https://doi.org/<the-publication-DOI>`

File: src/components/modules/publication.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  border-radius: 5px;
+  cursor: pointer;
+  color: var(--color-primary);
+  transition: color 250ms;
+  &:hover {
+    color: var(--color-primary-dark);
+  }
+`
+
+export const Publication = props => {
```
Comment (mbwatson) [inline src/components/modules/publication.js:20]: accessibility should be addressed here. check out the ARIA attributes `aria-controls`, `aria-expanded`, and `aria-labelled-by`.

**Review by mbwatson — COMMENTED** [review]
i just made a couple comments, but this is a great solution!

File: src/data/resources/publications/01-fabric-network-infrastructure.md (hunk 1 above)
Comment (yaxue1123) [reply src/data/resources/publications/01-fabric-network-infrastructure.md:5]: Good suggestion! I totally agree. I'll remove the doilink in data and modify the page accordingly.

File: src/components/modules/publication.js (hunk 2 above)
Comment (yaxue1123) [reply src/components/modules/publication.js:20]: Thanks for reminding about the accessibility. I'll add aria arributes.

**Pull request merged**